bug-1690385-pref-activity-stream-new-tab-default-content-release-85-87

```diff
@@ -0,0 +1,152 @@
+[experiment]
+end_date = '2020-04-13'
+
+segments = ["attributed", "non_attributed"]
+
+enrollment_period = 8
+
+reference_branch = "baseline-newtab-content"
+
+
+## data sources
+
+[data_sources]
+
+## data source: AS session pings data
+[data_sources.as_session_ping]
+from_expression = "(SELECT client_id, user_prefs, date(submission_timestamp) as submission_date FROM  `moz-fx-data-shared-prod.activity_stream.sessions`)"
+
+experiments_column_type = "native"
+
+
+
+
+## Segments
+
+[segments]
+
+[segments.attributed]
+select_expression = 'attr_source is not null'
+data_source = "npp"
+
+[segments.non_attributed]
+select_expression = 'attr_source is null'
```

Comment:
fwiw this segment will include clients where the distribution_id is not null, since they will not appear in the segment table, so those clients will have null values in all columns provided by the segment table after the join onto the enrollments -- not sure if that's what you want

Reply:
ahh that's good to know. 

that is, indeed my intention:
* attributed = non-null attribution source
* non-attributed = we have NPP without attribution + we have NPP but distribution + we don't have NPP

So it sounds like that select expression logic is executing AFTER we've joined the enrollments with the segements data source table, is that correct?

Reply:
Yes, that's a good model.